pkg/params/settings/config.go

```diff
@@ -93,6 +96,8 @@ type Settings struct {
 	CustomConsoleURL       string
 	CustomConsolePRdetail  string
 	CustomConsolePRTaskLog string
+
+	RememberOKToTest bool
```

Comment:
Can you also add unit test for `RememberOKToTest` in config_test.go

Reply:
done